arangod/RocksDBEngine/RocksDBReplicationTailing.cpp

```diff
@@ -86,11 +90,31 @@ TRI_replication_operation_e rocksutils::convertLogType(RocksDBLogType t) {
 
 /// WAL parser
 class WALParser : public rocksdb::WriteBatch::Handler {
+  
+  enum State : char { // mirrors code in RocksDBLogType
+    INVALID = 0,
+    COLLECTION_CREATE = '3',
+    COLLECTION_DROP = '4',
+    COLLECTION_RENAME = '5',
+    COLLECTION_CHANGE = '6',
+    INDEX_CREATE = '7',
+    INDEX_DROP = '8',
+    VIEW_CREATE = '9',
+    VIEW_DROP = ':',
+    VIEW_CHANGE = ';',
+    TRANSACTION = '<',
+    SINGLE_PUT = '?',
+    SINGLE_REMOVE = '@',
+    VIEW_RENAME = 'B',
+  };
+  
```

Comment:
they don't actually have to match the `RocksDBLogType` anymore, I just started out with that

Reply:
Okay, then let's remove the comment, and remove the explicit values for that matter. Otherwise it gives the impression that these are more intimately linked than they are.